ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml

```diff
@@ -199,15 +199,214 @@ presubmits:
     always_run: true
     branches:
     - master
-    context: ci/prow/unit
+    context: ci/prow/variant-benchmark
     decorate: true
     decoration_config:
       skip_cloning: true
     labels:
       ci-operator.openshift.io/prowgen-controlled: "true"
+      ci-operator.openshift.io/variant: variant
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-odo-master-unit
-    rerun_command: /test unit
+    name: pull-ci-openshift-odo-master-variant-benchmark
+    rerun_command: /test variant-benchmark
```

Comment:
```/test variant4.1-benchmark``` would be more precise as part test on cluster version is concerned. 

Just by updating the variant config in plugin.yaml like 
```
ci-operator/config/**/*master__variant*.yaml:
      name: ci-operator-master-configs
      additional_namespaces:
      - ci-stg
```
@stevekuznetsov Is it doable ?

Reply:
Using `master__4.1` was also fine -- do whichever you want

Reply:
Another pr https://github.com/openshift/release/pull/4904 sent to add upload config format of ```master__4.1```